install-deps.sh

```diff
@@ -286,6 +288,7 @@ if [ x$(uname)x = xFreeBSDx ]; then
         security/nss \
         security/krb5 \
         security/oath-toolkit \
+        security/cryptopp \
```

Comment:
as `WITH_SEASTAR=OFF` in do_freebsd.sh, we don't need this ? see also #20015

Reply:
@tchaikov 
Ah, missed that, during my period of $work. And I checked ceph.spec.in, were it still is mentioned. So I included it.
Will remove it from both install-deps and the port.

Reply:
@tchaikov 
Updated both files to the current state of affairs.